exercises/the-dice/metadata.yml

```diff
@@ -0,0 +1,2 @@
+---
+blurb: "Count number of unique pair of 6s afther rolling the dice"
```

Comment:
have you a source and source_url for this exercise, like other exercises have?

If it is a problem you made up yourself, you could do like https://github.com/exercism/x-common/blob/master/exercises/paasio/metadata.yml#L3 did.

Or if it is acquired from somewhere, you should enter the appropriate values in this file (important so we can know it is OK for us to use it)